tasks.py

```diff
@@ -54,7 +54,7 @@ def fmt(ctx, path=".", sort_=True, check=False, exclude=None):
         cmds.append("--check")
     if exclude:
         cmds.extend(["--exclude", exclude])
-    ctx.run(" ".join(cmds))
+    ctx.run(" ".join(cmds), echo=True)
```

Comment:
![Screen Shot 2022-08-01 at 4 36 57 PM](https://user-images.githubusercontent.com/13108583/182250927-0e15fa0b-fa40-4223-aa6f-16adccdfde05.png)

Reply:
Thank you!